app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -0,0 +1,8 @@
+// Override bootstrap variables here
+
+// Headings
+$font-size-h1: 2.323em;
+$font-size-h2: 1.793em;
+$font-size-h3: 1.383em;
+$font-size-h4: 1.215em;
```

Comment:
These font sizes aren't actually working...

<img width="491" alt="Screen Shot 2019-07-09 at 10 22 44 PM" src="https://user-images.githubusercontent.com/96776/60942689-608b9580-a298-11e9-8881-d3f20c9d7004.png">

We are explicitly setting various `h*` declarations, so maybe we should just set those there.  (the font-weight below does work, soooo)